utils/ethformatter.js

```diff
@@ -5,7 +5,12 @@ function formatAmount(amount) {
   var amountBN = new BigNumber(amount.toString());
   var amountStr = amountBN.toString(10);
   var amountAsEth = web3.utils.fromWei(amountStr, 'ether');
-  return amountAsEth.toString() + " ETH";
+
+  if (amount === 0) {
+    return "0 ETH (pending?)";

```

Comment:
Not sure this is necessary?

Reply:
I'm not sure either, but I was thinking showing a 0 fee would be confusing, and with that message we know it's because the transaction is still pending, but I'm OK with removing it or leaving the result blank.

Reply:
This is not only used for fees, but values in general.

Reply:
you're right, I'll revert this change